src/utils/signRelease.ts

```diff
@@ -191,3 +173,37 @@ function cidToString(cid: CID, base: string): string {
       throw Error(`Unknown CID base ${base}`);
   }
 }
+
+interface IpfsDagGet {
+  Name: string;
+  Size: number;
+  Hash: string;
+}
+
+/**
+ * REMOTE
+ * List items in CID using Gateway endpoint /dag/get
+ * IMPORTANT! ipfs.ls method not allowed in infura gateway, must be used dag-get
+ * @param ipfs
+ * @param hash
+ */
+export async function dagGet(ipfs: IPFSHTTPClient, hash: string): Promise<IPFSEntry[]> {
+  const files: IPFSEntry[] = [];
+  const hashSanitized = parseIpfsPath(hash);
+  const cid = CID.parse(hashSanitized);
+  const content = await ipfs.dag.get(cid);
+  const contentLinks: IpfsDagGet[] = content.value.Links;
+  if (!contentLinks) throw Error(`hash ${hashSanitized} does not contain links`);
+  // eslint-disable-next-line array-callback-return
+  contentLinks.map((link) => {
+    if (!cid) throw Error("Error getting cid");
```

Comment:
This check seems like it can happen outside of the map right? Then the `eslint-disable-next-line array-callback-return` is not necessary